tests/unit/nft_id_test.py

```
@@ -5,42 +5,54 @@
     assert NftId._from_proto(
         nft_id_proto=basic_types_pb2.NftID(
             token_ID=basic_types_pb2.TokenID(shardNum=0, realmNum=1, tokenNum=2),
-            serial_number=1234
+            serial_number=1234,
         )
     ).__eq__(nftid_constructor_test)
 
-    #return false
+    # return false
     with pytest.raises(TypeError):
         nftid_failed_constructor_tokenid1 = TokenId(shard=0, realm=1, num="A")
```

Ruff (0.14.14)

Local variable `nftid_failed_constructor_tokenid1` is assigned to but never used

Remove assignment to unused variable `nftid_failed_constructor_tokenid1`

(F841)